packages/aztec.js/test/proof/dividend/verifier.js

```diff
@@ -7,7 +7,7 @@ const { padLeft, randomHex } = require('web3-utils');
 const sinon = require('sinon');
 
 const { DividendProof } = require('../../../src/proof');
-const DividendVerifier = require('../../../src/proof/dividend/verifier');
+const DividendVerifier = require('../../../src/proof/epoch0/UTILITY/dividend/verifier');
```

Comment:
small, but I think it makes sense for the proof type to be a parent of an epoch:

```
- UTILITY
  - epoch0
  - epoch1
```
instead of 
```
- Epoch0
  - utility
  - balanced
- Epoch1
  - utility
```

Reply:
As discussed, leaving as is